vars/lambdaTests.groovy

```diff
@@ -0,0 +1,81 @@
+
+def call(Map config) {
+    library("tdr-jenkinslib")
+
+    def versionTag = "v${env.BUILD_NUMBER}"
+    def repo = "tdr-${config.libraryName}"
+
+    pipeline {
+        agent {
+            label "master"
+        }
+        parameters {
+            choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are building the front end for")
```

Comment:
Probably want to change the description for the STAGE parameter

Reply:
I've changed this one as well.